plugins/runtime/threads/threads.component.ts

```diff
@@ -167,19 +180,36 @@ namespace Runtime {
     disableThreadContentionMonitoring(): void {
       this.threadsService.disableThreadContentionMonitoring()
         .then(() => this.showThreadContentionMonitoringDisabledView());
-      }
-      
+    }
+
     showThreadContentionMonitoringDisabledView() {
       this.threadContentionMonitoringEnabled = false;
       this.toolbarActions[0] = this.enableThreadContentionMonitoringAction;
       this.$interval.cancel(this.intervalId);
     }
+    performThreadDump(): void {
```

Comment:
It's always required to add a blank line between two function/method definitions.